src/server/usage/classes/collector_set.js

```diff
@@ -117,19 +117,22 @@ export class CollectorSet {
       }
     };
 
-    return Object.keys(apiData).reduce((accum, currName) => {
-      const value = apiData[currName];
-
-      let newName = currName;
-      newName = snakeCase(newName);
-      newName = newName.replace(/^(1|5|15)_m/, '$1m'); // os.load.15m, os.load.5m, os.load.1m
-      newName = newName.replace('_in_bytes', '_bytes');
-      newName = newName.replace('_in_millis', '_ms');
-
-      return {
-        ...accum,
-        [newName]: getValueOrRecurse(value),
-      };
-    }, {});
+    if (Array.isArray(apiData)) {
+      return apiData.map(getValueOrRecurse);
+    } else {
```

Comment:
Since you are `return`ing from the `if` above, you don't necessarily need the `else` here. Somewhat related: https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#returnthrow-early-from-functions.